cloudfront/wellcomelibrary.org/edge-lambda/apiRedirects.csv

```diff
@@ -0,0 +1,14 @@
+sourceUrl,targetUrl
```

Comment:
Would it be worth an optional "description" field that has a human-readable description of the test case?

Reply:
Good idea.